std_msgs/Header header
float64[] rpm
float64[] current
uint8[] error_status